### PR TITLE
add sentry arguments

### PR DESCRIPTION
**Pull request by Donbur4156**
add optional args for sentry_sdk.init() to further personalize the sentry handler

## Pull Request Type

- [X] Feature addition
- [ ] Bugfix
- [ ] Documentation update
- [ ] Code refactor
- [ ] Tests improvement
- [ ] CI/CD pipeline enhancement
- [ ] Other: [Replace with a description]

## Description

Add possibility to pass individual arguments to sentry_sdk.init
Helpful for changing environment to "development" e.g.

## Changes

- add argument "sentry_args" to setup() in sentry.py
- add "**sentry_args" to sentry_sdk.init() inside setup function

## Related Issues

#1432 

## Test Scenarios

```python
sentry_args = { "environment": "development"}
client.load_extension('interactions.ext.sentry', token=SENTRY_TOKEN, sentry_args=sentry_args)
```

## Python Compatibility

- [X] I've ensured my code works on Python `3.10.x`
- [ ] I've ensured my code works on Python `3.11.x`

## Checklist

- [ ] I've run the `pre-commit` code linter over all edited files
- [X] I've tested my changes on supported Python versions
- [ ] I've added tests for my code, if applicable
- [ ] I've updated / added  documentation, where applicable

### CODE REVIEW

File: interactions/ext/sentry.py [hunk 1]
```diff
@@ -91,12 +91,13 @@ def setup(
     bot: Client,
     token: str = None,
     filter: Optional[Callable[[dict[str, Any], dict[str, Any]], Optional[dict[str, Any]]]] = None,
+    sentry_args: dict[str, Any] = None,
```
Comment (silasary) [inline interactions/ext/sentry.py]: I'd prefer this be **kwargs, as proposed in the initial issue

**Review by silasary — CHANGES_REQUESTED** [review]

File: interactions/ext/sentry.py (hunk 1 above)
Comment (Donbur4156) [reply interactions/ext/sentry.py]: Yeah, this would work too.
The idea was to explicit pass arguments into the init()

Comment (silasary) [reply interactions/ext/sentry.py]: Asking people to enclose parameters in a dict doesn't have any benefit, makes the UX worse and the code less readable.

Comment (Donbur4156) [reply interactions/ext/sentry.py]: Ah ok. Then **kwargs

**Review by silasary — APPROVED** [review]

**Pull request merged**